server/api-manager.js

```diff
@@ -885,7 +889,7 @@ export default class MindbodyQueries {
   getStaff(parameters) {
     var query = QueryString.stringify(parameters);
     var request = new MindbodyRequest(
-      URL_STAFF + "/staff" + query,
+      URL_STAFF + "/staff?" + query,
```

Comment:
Without this "?", the query string was not being formed right. Am I missing something about QueryString.stringify??? If this change is correct, then it should be applied to every var request that this file builds, because they're all done the same way. I just changed getStaff because that's the only one I used for now.